[2022-10-27 00:53:42,888] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: spark_job_dag.docker_command scheduled__2022-10-26T00:53:28.259836+00:00 [queued]>
[2022-10-27 00:53:42,897] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: spark_job_dag.docker_command scheduled__2022-10-26T00:53:28.259836+00:00 [queued]>
[2022-10-27 00:53:42,897] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-27 00:53:42,898] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-27 00:53:42,898] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-27 00:53:42,911] {taskinstance.py:1377} INFO - Executing <Task(DockerOperator): docker_command> on 2022-10-26 00:53:28.259836+00:00
[2022-10-27 00:53:42,916] {standard_task_runner.py:52} INFO - Started process 238 to run task
[2022-10-27 00:53:42,919] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'spark_job_dag', 'docker_command', 'scheduled__2022-10-26T00:53:28.259836+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/spark_job_dag.py', '--cfg-path', '/tmp/tmp5crtingd', '--error-file', '/tmp/tmpe0dwo2d1']
[2022-10-27 00:53:42,920] {standard_task_runner.py:80} INFO - Job 10: Subtask docker_command
[2022-10-27 00:53:42,976] {task_command.py:369} INFO - Running <TaskInstance: spark_job_dag.docker_command scheduled__2022-10-26T00:53:28.259836+00:00 [running]> on host abbfe66d2e6a
[2022-10-27 00:53:43,050] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=spark_job_dag
AIRFLOW_CTX_TASK_ID=docker_command
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T00:53:28.259836+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T00:53:28.259836+00:00
[2022-10-27 00:53:43,083] {docker.py:371} INFO - Pulling docker image bde2020/spark-master:latest
[2022-10-27 00:53:45,268] {docker.py:385} INFO - latest: Pulling from bde2020/spark-master
[2022-10-27 00:53:45,269] {docker.py:385} INFO - 396c31837116: Pulling fs layer
[2022-10-27 00:53:45,270] {docker.py:385} INFO - bb59ba756ca3: Pulling fs layer
[2022-10-27 00:53:45,271] {docker.py:385} INFO - 52d9f65633a2: Pulling fs layer
[2022-10-27 00:53:45,271] {docker.py:385} INFO - 7cdbc775d228: Pulling fs layer
[2022-10-27 00:53:45,272] {docker.py:385} INFO - 1832b9aadd75: Pulling fs layer
[2022-10-27 00:53:45,273] {docker.py:385} INFO - 4f4fb700ef54: Pulling fs layer
[2022-10-27 00:53:45,273] {docker.py:385} INFO - 616308593c75: Pulling fs layer
[2022-10-27 00:53:45,274] {docker.py:385} INFO - 7cdbc775d228: Waiting
[2022-10-27 00:53:45,275] {docker.py:385} INFO - 1832b9aadd75: Waiting
[2022-10-27 00:53:45,275] {docker.py:385} INFO - 4f4fb700ef54: Waiting
[2022-10-27 00:53:45,276] {docker.py:385} INFO - 616308593c75: Waiting
[2022-10-27 00:53:45,987] {docker.py:385} INFO - 396c31837116: Downloading
[2022-10-27 00:53:45,988] {docker.py:385} INFO - 52d9f65633a2: Downloading
[2022-10-27 00:53:45,989] {docker.py:385} INFO - 52d9f65633a2: Download complete
[2022-10-27 00:53:45,989] {docker.py:385} INFO - bb59ba756ca3: Downloading
[2022-10-27 00:53:45,990] {docker.py:385} INFO - bb59ba756ca3: Verifying Checksum
[2022-10-27 00:53:45,991] {docker.py:385} INFO - bb59ba756ca3: Download complete
[2022-10-27 00:53:46,579] {docker.py:385} INFO - 396c31837116: Download complete
[2022-10-27 00:53:46,579] {docker.py:385} INFO - 396c31837116: Extracting
[2022-10-27 00:53:46,792] {docker.py:385} INFO - 396c31837116: Pull complete
[2022-10-27 00:53:46,793] {docker.py:385} INFO - bb59ba756ca3: Extracting
[2022-10-27 00:53:46,793] {docker.py:385} INFO - 7cdbc775d228: Downloading
[2022-10-27 00:53:46,794] {docker.py:385} INFO - 7cdbc775d228: Download complete
[2022-10-27 00:53:46,795] {docker.py:385} INFO - bb59ba756ca3: Pull complete
[2022-10-27 00:53:46,795] {docker.py:385} INFO - 52d9f65633a2: Extracting
[2022-10-27 00:53:46,796] {docker.py:385} INFO - 52d9f65633a2: Pull complete
[2022-10-27 00:53:46,796] {docker.py:385} INFO - 7cdbc775d228: Extracting
[2022-10-27 00:53:47,003] {docker.py:385} INFO - 7cdbc775d228: Pull complete
[2022-10-27 00:53:47,005] {docker.py:385} INFO - 1832b9aadd75: Downloading
[2022-10-27 00:53:47,227] {docker.py:385} INFO - 4f4fb700ef54: Downloading
[2022-10-27 00:53:47,227] {docker.py:385} INFO - 4f4fb700ef54: Verifying Checksum
[2022-10-27 00:53:47,228] {docker.py:385} INFO - 4f4fb700ef54: Download complete
[2022-10-27 00:53:47,667] {docker.py:385} INFO - 616308593c75: Downloading
[2022-10-27 00:53:47,667] {docker.py:385} INFO - 616308593c75: Verifying Checksum
[2022-10-27 00:53:47,668] {docker.py:385} INFO - 616308593c75: Download complete
[2022-10-27 00:54:00,996] {docker.py:385} INFO - 1832b9aadd75: Verifying Checksum
[2022-10-27 00:54:00,997] {docker.py:385} INFO - 1832b9aadd75: Download complete
[2022-10-27 00:54:00,998] {docker.py:385} INFO - 1832b9aadd75: Extracting
[2022-10-27 00:54:12,802] {docker.py:385} INFO - 1832b9aadd75: Pull complete
[2022-10-27 00:54:12,804] {docker.py:385} INFO - 4f4fb700ef54: Extracting
[2022-10-27 00:54:12,805] {docker.py:385} INFO - 4f4fb700ef54: Pull complete
[2022-10-27 00:54:12,806] {docker.py:385} INFO - 616308593c75: Extracting
[2022-10-27 00:54:12,807] {docker.py:385} INFO - 616308593c75: Pull complete
[2022-10-27 00:54:12,808] {docker.py:380} INFO - Digest: sha256:3fe3e2c13a3d1f889b5e6127851b56734c3dbd0c268699e5fd3273631973c418
[2022-10-27 00:54:12,808] {docker.py:380} INFO - Status: Downloaded newer image for bde2020/spark-master:latest
[2022-10-27 00:54:12,809] {docker.py:247} INFO - Starting docker container from image bde2020/spark-master:latest
[2022-10-27 00:54:12,829] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/api/client.py", line 268, in _raise_for_status
    response.raise_for_status()
  File "/home/airflow/.local/lib/python3.7/site-packages/requests/models.py", line 960, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://docker-socket-proxy:2375/v1.37/containers/create

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/docker/operators/docker.py", line 387, in execute
    return self._run_image()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/docker/operators/docker.py", line 254, in _run_image
    return self._run_image_with_mounts(self.mounts + [tmp_mount], add_tmp_variable=True)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/docker/operators/docker.py", line 298, in _run_image_with_mounts
    tty=self.tty,
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/api/container.py", line 428, in create_container
    return self.create_container_from_config(config, name)
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/api/container.py", line 439, in create_container_from_config
    return self._result(res, True)
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/api/client.py", line 274, in _result
    self._raise_for_status(response)
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/api/client.py", line 270, in _raise_for_status
    raise create_api_error_from_http_exception(e)
  File "/home/airflow/.local/lib/python3.7/site-packages/docker/errors.py", line 31, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation)
docker.errors.APIError: 500 Server Error for http://docker-socket-proxy:2375/v1.37/containers/create: Internal Server Error ("json: cannot unmarshal string into Go struct field HostConfig.HostConfig.Mounts of type mount.Mount")
[2022-10-27 00:54:12,855] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=spark_job_dag, task_id=docker_command, execution_date=20221026T005328, start_date=20221027T005342, end_date=20221027T005412
[2022-10-27 00:54:12,877] {standard_task_runner.py:97} ERROR - Failed to execute job 10 for task docker_command (500 Server Error for http://docker-socket-proxy:2375/v1.37/containers/create: Internal Server Error ("json: cannot unmarshal string into Go struct field HostConfig.HostConfig.Mounts of type mount.Mount"); 238)
[2022-10-27 00:54:12,915] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-10-27 00:54:12,952] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
